src/commands/init/index.ts

```diff
@@ -75,6 +76,18 @@ export const handleInitializeCli = async (cmdInput: InitProjectInput) => {
       }
     }
 
+    /**
+     * Analytics
+     */
+    let allowAnalytics = false
+
+    if (!cmdInput.ignoreAnalytics) {
+      allowAnalytics = await confirm({
+        message: `Do you allow INFINIT CLI to send usage data to help improve the tool? (Can be changed later in the config file)`,
```

Comment:
```suggestion
        message: `Do you allow INFINIT CLI to send usage data to help improve the tool? (This can be changed later in the config file)`,
```